docker/default/etc/jvm.config

```diff
@@ -0,0 +1,18 @@
+-server
+-XX:InitialRAMPercentage=80
+-XX:MaxRAMPercentage=80
+-XX:G1HeapRegionSize=32M
+-XX:+ExplicitGCInvokesConcurrent
+-XX:+HeapDumpOnOutOfMemoryError
+-XX:+ExitOnOutOfMemoryError
+-XX:-OmitStackTraceInFastThrow
+-XX:ReservedCodeCacheSize=256M
+-XX:PerMethodRecompilationCutoff=10000
+-XX:PerBytecodeRecompilationCutoff=10000
+-Djdk.attach.allowAttachSelf=true
+-Djdk.nio.maxCachedBufferSize=2000000
+# Reduce starvation of threads by GClocker, recommend to set about the number of cpu cores (JDK-8192647)
+-XX:+UnlockDiagnosticVMOptions
+-XX:GCLockerRetryAllocationCount=32
```

Comment:
This config has been removed in java22 / trino 447

we should get this config from the [latest trino's jvm.config file.](https://github.com/trinodb/trino/blob/master/core/docker/default/etc/jvm.config) 
can you update it?